### PR TITLE
freqofwords2

### PR DESCRIPTION
**Pull request by ana-bharadwaj**

### CODE REVIEW

**Comment by Tejusbharadwaj** [comment]
Well there is a small error. Try running it for this `freq.CountWords("as bat hat js", "as hat bat gsg asd bat hat");`

File: src/FreqOfWords2.java [hunk 1]
```diff
@@ -12,15 +12,32 @@
...
+            for (String words : s2) {
+                if (map.containsKey(words)) {
+                    int ke = map.get(word);
+                    map.put(words, ke + 1);
+                }
+
+            }
+
+        }
+        for (String name : map.keySet()) {
```
Comment (Tejusbharadwaj) [inline src/FreqOfWords2.java:37]: Printing this way is perfectly fine, but I just wanted to let you know there is another way to print a map in java:
```
 map.forEach((key, value) -> System.out.println(key + "-->" + value));
```
This is just a FYI.

File: src/FreqOfWords2.java [hunk 2]
```diff
@@ -12,15 +12,32 @@
...
+import java.util.HashMap;
 public class FreqOfWords2 {
     public static void main(String[] args) {
         FreqOfWords2 freq = new FreqOfWords2();
         freq.CountWords("My name is foo", "My passtime is foo and my pet is foo and it is the best");
     }
 
     public void CountWords(String a, String b) {
-        // Your code goes here...
+        String[] s1 = a.split("\\s");
```
Comment (Tejusbharadwaj) [inline src/FreqOfWords2.java:23]: Can you tell me what is the time complexity of this program?

**Review by Tejusbharadwaj — CHANGES_REQUESTED** [review]

**Comment by malvikakumar** [comment]
What output did you get for the program?

**Pull request closed without merging**